releasing.md

```diff
@@ -0,0 +1,9 @@
+# Release procedure
```

Comment:
can we put this as a section in a file with a well-known name, like CONTRIBUTING.md or HACKING.md or something?

Reply:
Moved to docs directory.